sql/activity_stream_bi/impression_stats_flat_v1/query.sql

```diff
@@ -20,6 +20,7 @@ SELECT
   user_prefs,
   release_channel,
   IFNULL(shield_id, 'n/a') AS shield_id,
+  ANY_VALUE(experiments),
```

Comment:
I'm assuming here that all the rows we get will have identical values for `experiments`, since we're aggregating across per-tile rows all associated with the same `impression_stats` ping.